### DISCUSSION TITLE
Possible hide default native filter section in standalone mode /  embedded in iframe

### DISCUSSION THREAD

**Discussion by rafalkasa** [apache/superset#18361] [Q&A / Help] [answered]
## Screenshot

![image](https://user-images.githubusercontent.com/19885116/146693239-046afde8-7ad1-40fb-b87b-37ec334685fd.png)

## Description

Could you let me know how I can collapse Dashboard Native Filter section when I embedded dashboard in iframe ?

## Possible solution

It will be nice if it will be additional parameters as it is eg. for Power BI (https://community.powerbi.com/t5/Service/Hiding-Report-Level-filters-in-the-Iframe/m-p/64756)

**filterPaneEnabled=false**

`http://localhost:8088/superset/dashboard/12/?standalone=2&filterPaneEnabled=false`

**Comment by rusackas** [comment]
Pinging @suddjian @lilykuang @amitmiran137 to see if/how they've considered this need in particular.

**Comment by villebro** [comment]
IIRC it can be hidden/collapsed with a URL param. Let me see if I can find it and add a note in the docs.

**Comment by amitmiran137** [comment]
I specifically use the **show_native_filters** property under the dashboard metadata to decide per dashboard whether to show and use the native filter left side panel

**Reply by hbruch** [reply]
@villebro could you find the URL param's name? I'd like create links to embed a dashboard in an iframe without native_filters, but have them visible in other cases, so a metadata toggle is not sufficient

**Reply by isaias-b** [reply]
funny i searched this up and found your suggestion 🤣

**Comment by villebro** [comment, marked as answer]
@hbruch I opened a PR to fix this and add docs: #18741

**Reply by srinisubramanian** [reply]
@villebro Any idea when this will be  merged?

**Reply by rafalkasa** [reply]
Exactly good question, any plans ?

**Reply by villebro** [reply]
@srinisubramanian and @rafalkasa thanks for the reminder - it's now merged and I've tagged it to be included in 1.5.0

**Comment by dreamact** [comment]
Please update DASHBOARD_CROSS_FILTERS item in config.py file to False
"DASHBOARD_CROSS_FILTERS": True, (No)  ===> **"DASHBOARD_CROSS_FILTERS": Fasle (Good)**
It worked fine.
